src/lib/button-toggle/button-toggle.md

```diff
@@ -18,5 +18,12 @@ and `ngModel` is not supported.
 The button-toggles will present themselves as either checkboxes or radio-buttons based on the 
 presence of the `multiple` attribute. 
 
+For button toggles with icons only, Each button toggle should be marked with meaningful label by 
```

Comment:
"each" shouldn't be capitalized. Also here and in a few of the other files "with meaningful label" -> "with a meaningful label".